c7n/resources/ec2.py

```diff
@@ -652,6 +652,83 @@ def process_volume_set(self, resource):
                     response['SnapshotId']],
                 Tags=tags)
 
+@actions.register('rename-tag')
+class RenameTag(BaseAction):
+
+    schema = type_schema(
+        'rename-tag',
+        map={
```

Comment:
sorry missed this key earlier, but still puzzled, why nest the values in a map if your just processing a single key/value versus making them top level